src/main/java/uk/gov/hmcts/divorce/systemupdate/service/task/GenerateApplyForFinalOrderDocument.java

```diff
@@ -0,0 +1,56 @@
+package uk.gov.hmcts.divorce.systemupdate.service.task;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.divorce.divorcecase.model.Applicant;
+import uk.gov.hmcts.divorce.divorcecase.model.CaseData;
+import uk.gov.hmcts.divorce.document.CaseDataDocumentService;
+import uk.gov.hmcts.divorce.notification.CommonContent;
+
+import java.time.Clock;
+import java.time.LocalDateTime;
+
+import static uk.gov.hmcts.divorce.caseworker.service.task.util.FileNameUtil.formatDocumentName;
+import static uk.gov.hmcts.divorce.document.DocumentConstants.FINAL_ORDER_CAN_APPLY_DOCUMENT_NAME;
+import static uk.gov.hmcts.divorce.document.DocumentConstants.FINAL_ORDER_CAN_APPLY_TEMPLATE_ID;
+import static uk.gov.hmcts.divorce.document.model.DocumentType.FINAL_ORDER_CAN_APPLY;
+
+@Component
+@Slf4j
+public class GenerateApplyForFinalOrderDocument {
+
+    public static final String FIRST_NAME = "firstName";
+    public static final String LAST_NAME = "lastName";
```

Comment:
Are these static FIRST_NAME & LAST_NAME vars used in this class?

Reply:
removed the unused constants, thanks